.circleci/config.yml

```diff
@@ -214,9 +214,9 @@ jobs:
           command: |
               export PATH=$HOME/miniconda/bin:$PATH
               . activate habitat; cd habitat-api
-              python setup.py test
               python setup.py develop --all
-              python -u habitat_baselines/train_ppo.py --log-file "train.log" --checkpoint-folder "data/checkpoints" --sim-gpu-id 0 --pth-gpu-id 0 --num-processes 1 --num-mini-batch 1 --num-updates 10
+              python setup.py test
+              python -u habitat_baselines/run.py --exp-config habitat_baselines/config/pointnav/ppo_train_test.yaml --run-type train
```

Comment:
hmm...
Rebased on or merged from master branch, and it shows in my branch as change.
Strange.
How can I get rid of it?

Reply:
Did you merge with master or did you just cherry-pick the commit of #118?

Reply:
@erikwijmans : No, I did not do cherry-pick. I did rebase or merge master though. Very strange.

Reply:
This can be easily fixed with a manual merge (squash everything into a single commit and then remove this change). I can do that once we all approve :)